grpc-codegen/src/main/scala/monix/grpc/codegen/package.scala

```diff
@@ -0,0 +1,43 @@
+package monix.grpc
+
+import protocbridge.{Artifact, SandboxedJvmGenerator}
+import scalapb.GeneratorOption
+import scalapb.GeneratorOption.FlatPackage
+
+/**
+  * Copyright (C) 08.11.21 - REstore NV
+  */
+
+package object codegen {
+  val SandboxedGenerator = SandboxedJvmGenerator.forModule(
+    "scala",
+    Artifact(
+      "me.vican.jorge",
+      "monix-grpc-codegen_2.12",
+      BuildInfo.version
+    ),
+    "scalapb.ScalaPbCodeGenerator$",
+    GrpcCodeGenerator.suggestedDependencies
+  )
+
+  def apply(options: Set[GeneratorOption]): (SandboxedJvmGenerator, Seq[String]) =
```

Comment:
Looks like we're going to support other options in the future, I suggest we use the builder pattern instead and remove the need for all these redundant `apply` methods with default arguments. Will simplify the API and will remove the need to return a tuple.

Reply:
The tupple is what we have to return to the scalapb plugin part of sbt. So we can't go around that. (i.e. it is passed like strings)